ssl/statem/statem_clnt.c

```diff
@@ -2454,7 +2450,6 @@ MSG_PROCESS_RETURN tls_process_new_session_ticket(SSL *s, PACKET *pkt)
     if (ticklen == 0)
         return MSG_PROCESS_CONTINUE_READING;
 
-    /* TODO(TLS1.3): Is this a suitable test for TLS1.3? */
```

Comment:
I guess we are decreeing that this is a suitable test for TLS1.3 by virtue of having a given SSL_SESSION object (for TLS1.3) always represent the most recently received ticket from the server, so that the SSL_SESSION would correspond to (e.g.) different keying material at different points in time.
Should we have a broader discussion about whether it's better to have an invariant that SSL_SESSION and tickets are a 1-1 correspondence?  (I have not spent much time thinking about this issue, and will defer if you have done so already.)